lib/astronoby/bodies/sun.rb

```diff
@@ -16,7 +16,7 @@ def ecliptic_coordinates
       Coordinates::Ecliptic.new(
         latitude: Angle.as_degrees(0),
```

Comment:
Also out of scope for this PR, since zero doesn't care about units, I wonder if having an `Angle.zero` constructor would be a slight win from a readability perspective. That way readers don't start wondering "why are we using degrees here?"

Reply:
I love it.
To me, it makes sense in this PR, so I added it 👍